### PR TITLE
Zeek added to Attack Range

### PR DESCRIPTION
**Pull request by davisshannon**
Added in Zeek functionality to Attack Range.

Changed AWS instance types to be defined in attack-range.conf

Added ability to not have Splunk server spun up (helps with testing other instances as you don't have to wait for Splunk to complete)

### CODE REVIEW

**Comment by ghoto** [comment]
hey @davisshannon thanks for the PR. I see multiple changes that seem not related to the bare functionality to enable Zeek to attack range. In particular configurable instance types in attack_range.conf, personal keys, etc. Would you mind cleaning a bit this PR to include only the bare changes to enable Zeek?

File: ansible/playbooks/splunk_server.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - hosts: all
-  gather_facts: False
+  gather_facts: True
```
Comment (ghoto) [inline ansible/playbooks/splunk_server.yml:2]: - [] perhaps keep original

File: attack_range.conf [hunk 2]
```diff
@@ -10,15 +10,15 @@ log_level = INFO
 
 
 [range_settings]
-key_name = attack-range-key-pair
+key_name = attack-range-shannond-oregon
```
Comment (ghoto) [inline attack_range.conf:13]: This file needs to be cleared up.

File: attack_range.conf [hunk 3]
```diff
@@ -268,10 +287,19 @@ windows_domain_controller_os = Windows_Server-2016-English-Full-Base-*
...
 
+windows_domain_controller_zeek_capture = 0
+#specify if you want to capture wire data from the Windows Domain Controller
+# possible values: 1, 0
+
 
 [windows_server]
 # customize the windows server
 
+windows_server_instance_type = m5.2xlarge
```
Comment (ghoto) [inline attack_range.conf:298]: This should be its own PR. Same for the ones that follow down

File: attack_range.conf [hunk 4]
```diff
@@ -306,6 +346,26 @@ windows_client_os = import-ami-0e5a092cc20dbb992
...
+# possible values: 1, 0
+
+[zeek_sensor]
+# customize the zeek sensor
+
+zeek_sensor_instance_type = m5.2xlarge
+# specify the instance type for the zeek sensor
+# you must use an AWS Nitro instance type (m5.2xlarge)
+
+splunk_uf_linux_deb_url = https://download.splunk.com/products/universalforwarder/releases/8.0.5/linux/splunkforwarder-8.0.5-a1a6394cc5ae-linux-2.6-amd64.deb
```
Comment (ghoto) [inline attack_range.conf:364]: Not sure if we want this in the configuration.

File: attack_range.conf [hunk 5]
```diff
@@ -306,6 +346,26 @@ windows_client_os = import-ami-0e5a092cc20dbb992
...
+# Specify the download URL of the Splunk Linux (debian based) universal forwarder
+
+splunk_uf_binary = splunkforwarder-8.0.5-a1a6394cc5ae-linux-2.6-amd64.deb
```
Comment (ghoto) [inline attack_range.conf:367]: Not sure if we want this in the configuration.

File: attack_range.conf [hunk 6]
```diff
@@ -220,11 +223,15 @@ caldera_password = I-l1ke-Attack-Range!
 [environment]
 # specify your Attack Range environment by enabling (1) or disabling (0) machines
 
+splunk_server = 1
```
Comment (ghoto) [inline attack_range.conf:226]: Not sure if we want optional splunk_server

File: attack_range.conf (hunk 4 above)
Comment (davisshannon) [reply attack_range.conf:364]: This is needed for the UF that runs on the Zeek sensor.

File: attack_range.conf (hunk 5 above)
Comment (davisshannon) [reply attack_range.conf:367]: This is also needed for the UF that runs on the Zeek sensor

File: attack_range.conf (hunk 2 above)
Comment (davisshannon) [reply attack_range.conf:13]: Oops.  Reverting the keys back to original attack-range.conf ones.

File: attack_range.conf (hunk 3 above)
Comment (davisshannon) [reply attack_range.conf:298]: I'll change the ones back that don't have any bearing on the capture interfaces.  The ones for the Windows hosts need to be defined as Nitro instances, or else the VPC mirroring won't work.  I've put them into the Zeek config block for users to modify there if that makes more sense?

File: attack_range.conf (hunk 6 above)
Comment (davisshannon) [reply attack_range.conf:226]: Cool, I've taken out the optional bits for the Splunk server so that it always gets built.

**Comment by davisshannon** [comment]
Creating another PR as I didn't have the latest develop code.
